framework/include/utils/InputParameterWarehouse.h

```diff
@@ -75,7 +107,7 @@ class InputParameterWarehouse : public Warehouse<InputParameters>
    * This method is private, because only the factories that are creating objects should be
    * able to call this method.
    */
-  InputParameters & addInputParameters(const std::string & long_name, InputParameters parameters, THREAD_ID tid = 0);
+  InputParameters & addInputParameters(std::string name, InputParameters parameters, THREAD_ID tid = 0);
```

Comment:
It can be now, at one point I was just prefixing the supplied name with base (e.g., Kernel::name).